inbox/models/category.py

```diff
@@ -39,21 +48,29 @@ def API_OBJECT_NAME(self):
 
     # STOPSHIP(emfree): need to index properly for API filtering performance.
     name = Column(String(MAX_INDEXABLE_LENGTH), nullable=False, default='')
-    display_name = Column(String(MAX_INDEXABLE_LENGTH,
-                                 collation='utf8mb4_bin'), nullable=False)
+    display_name = Column(CategoryNameType(MAX_INDEXABLE_LENGTH,
```

Comment:
If you wanted to make this even simpler, you could override `__init__`  too and pass in `MAX_INDEXABLE_LENGTH` from within `CategoryNameType()`'s constructor to the superclass—that would eliminate any chance of accidentally using a different length anywhere.